examples/CMakeLists.txt

```diff
@@ -136,6 +136,11 @@ add_executable( 207-Rpt-TeamCityReporter ${EXAMPLES_DIR}/207-Rpt-TeamCityReporte
 #    add_executable( ${name} ${EXAMPLES_DIR}/${name}.cpp $<TARGET_OBJECTS:CatchMain> ${HEADER_DIR}/catch2/catch.hpp )
 #endforeach()
 
+if (CATCH_BUILD_STATIC_LIBRARY)
+    add_executable( MainFromStaticLib ${EXAMPLES_DIR}/100-Fix-Section.cpp )
+    target_link_libraries( MainFromStaticLib Catch2WithMain )
+endif()
```

Comment:
Why have these been added? What do they test?

Reply:
This test creates an executable that uses `main()` from the static lib. It allows to see whether the default reporter option is respected. Please see the description of the PR for the details.

The test is not automated though - one has to manually assess its output to see if the right reporter was used. As I ded.

Reply:
> The test is not automated though - one has to manually assess its output to see if the right reporter was used.

Right, that's the problem. If the test is not run during CI, then it effectively does not exist in the long term.

This change is kinda annoying to test, because it is a CMake config option, but not impossible. What would need to happen is to write a (Python, for cross-platform compatibility) script that configures a separate CMake build, builds it and checks the output afterwards -- this script can then be registered with CTest as a normal test.

Or, given how simple this change is, just leave it untested -- especially since it won't directly translate to the devel branch, which needs to handle this a bit differently.

Reply:
Ok, I see your point. I removed the test.